src/controller/CommissioningDelegate.h

```diff
@@ -105,17 +149,33 @@ class CommissioningDelegate
     virtual ~CommissioningDelegate(){};
     struct CommissioningReport
     {
+        CommissioningReport(CommissioningStage stage) : stageCompleted(stage) {}
         CommissioningStage stageCompleted;
         // TODO: Add other things the delegate needs to know.
         union
         {
```

Comment:
We have a c++14-compatible variant that can do this in a type-safe manner.  See src/lib/support/Variant.h.

Reply:
Didn't know about that - thanks. Would you mind if I made an issue and addressed this in a follow up PR? It would keep the change for that more contained.

Reply:
Yes, that's fine.

Reply:
Actually, never mind - had to wait for CI so I just changed it here. Wasn't as much change as I thought it would be.